lib/kamal/secrets/adapters/enpass.rb

```diff
@@ -0,0 +1,62 @@
+require "open3"
+
+##
+# Enpass is different from most password managers, in a way that it's offline. A path to a vault is treated as account.
+#
+# Pass it like so: `kamal secrets fetch --adapter enpass --account /Users/YOUR_USERNAME/Library/Containers/in.sinew.Enpass-Desktop/Data/Documents/Vaults/primary --from MY_PROD_SERVER`
+class Kamal::Secrets::Adapters::Enpass < Kamal::Secrets::Adapters::Base
+  private
+    def login(account)
+      # There is no concept of session in enpass-cli
+      true
+    end
+
+    def fetch_secrets(secrets, account:, session:)
+      secrets_titles = fetch_secret_titles(secrets)
+
+      # Enpass outputs result as stderr, I did not find a way to stub backticks and output to stderr. Open3 did the job.
+      _stdout, stderr, status = Open3.capture3("enpass-cli -vault #{account.shellescape} show #{secrets_titles.map(&:shellescape).join(" ")}")
```

Comment:
There no account here, so we probably:

1. Avoid passing an account at all
2. Use the from field to specify the vault

The base adapter just appends the from field and secrets together and passes them into `fetch_secrets`. 

That's not going to work here since the separator it uses is a `/`, so I think it should be ok to replace the `fetch` method for this adapter and have it pass the from field separately. 

We can refactor that to be neater later once all the other new adapter PRs have landed to avoid messing with them.

Reply:
Yes, if I could override the `fetch`, that would be cleaner.

Reply:
@djmb The `-account` option is marked as required in `Kamal::Cli::Secrets` (`option :account, type: :string, required: true`). If I want to not pass it at all, I need to make it optional. Is this a good idea to do it, just because of the way Enpass works? For other password managers it does make sense to have it required.

I think it's more practical to treat the Vault path as account for Enpass. What do you think?

Another option is to pass some dummy value as account and just not use it, but I'm also not sure about it.

Reply:
I went ahead and make `--account` optional and passed vault in `--from` 327d716